data/json/items/generic.json

```diff
@@ -2793,6 +2793,93 @@
     "weight": "774 g",
     "volume": "1750 ml",
     "flags": [ "SLEEP_AID" ],
+    "variant_type": "generic",
+    "variants": [
+      {
+        "id": "bodypillow_generic",
+        "name": { "str": "anime body pillow" },
+        "description": "A big, body-sized pillow with a print of an anime character on the front and their scantily clad version on the back.",
+        "weight": 1
+      },
+      {
+        "id": "bodypillow_azurlane",
+        "name": { "str": "ship body pillow" },
+        "description": "A big, body-sized pillow with a print of an aircraft carrier drawn on both sides.  The one on the back is stripped of its armor plating, how naughty.",
+        "weight": 1
+      },
+      {
+        "id": "bodypillow_gfl",
+        "name": { "str": "gun body pillow" },
+        "description": "A big, body-sized pillow with a print of a fully-modded tacticool M4 SOPMOD II on both sides.  The rifle firing a grenade somehow looks suggestive.",
+        "weight": 1
+      },
+      {
+        "id": "bodypillow_boston",
+        "name": { "str": "Boston-Chan body pillow" },
+        "description": "A big, body-sized pillow with a print of a blue-haired Boston-Chan, the mascot of Boston AnimeCon on both sides.  This seems to be official merch, and the art on both sides is quite tame.",
+        "weight": 2
+      },
+      {
+        "id": "bodypillow_boston_lewd",
+        "name": { "str": "Boston-Chan body pillow" },
+        "description": "A big, body-sized pillow with a print of a blue-haired Boston-Chan, the mascot of Boston AnimeCon on both sides.  It's highly unlikely the convention would officially endorse such a lewd depiction of her.",
+        "weight": 1
+      },
+      {
+        "id": "bodypillow_survivor",
+        "name": { "str": "survivor body pillow" },
+        "description": "A big, body-sized pillow with an anime-style print of Bear Grillings, Ultimate Survivor on both sides.  On one side he's holding a pneumatic air rifle, and on the other he is clad in nothing but duct tape.  The pillow is stylized to look like Kevlar.",
```

Comment:
```suggestion
        "description": "A big, body-sized pillow with an anime-style print of Bear Grillings, Ultimate Survivor, on both sides.  On one side he's holding a pneumatic air rifle, and on the other he is clad in nothing but duct tape.  The pillow is stylized to look like Kevlar.",
```

Reply:
In this case Ultimate Survivor is just a title rather than a description though so the comma isn't necessary.